.github/workflows/docs.yml

```diff
@@ -3,7 +3,17 @@ name: Build docs
 on:
   push:
     branches: [master]
+    paths-ignore:
+      - "tests/**"
+      - "docker/**"
+      - "assets/**"
+      - ".github/workflows/**"
   pull_request:
+    paths-ignore:
+      - "tests/**"
+      - "docker/**"
+      - "assets/**"
+      - ".github/workflows/**"
```

Comment:
Yeah, I was thinking to remove some workflows? but it going to be too verbose and messy (we also have some scripts there etc), so let's remove it, since we want to run docs workflow when `worlflows/docs.tml` changes as well.

```suggestion
```